docs/superpowers/specs/2026-03-10-saveresized-cascade-benchmark-design.md

```
@@ -0,0 +1,136 @@
+Add to `project/plugins.sbt`:
+```scala
+addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.4.7")
+```
+
+Enable in `build.sbt` on the root project:
+```scala
+.enablePlugins(..., JmhPlugin)
+```
+
+Run benchmarks with:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)